.github/workflows/test-all-packages.yml

```
@@ -14,10 +14,21 @@ on:
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
+
 env:
+  NEW_NODE_LTS_VERSION: '20.x'
```

would this also work?
```suggestion
  NEW_NODE_LTS_VERSION: 20
```

---

Yes it does. Done